ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/websocket/WebSocketService.java

```diff
@@ -94,6 +95,8 @@ private WebSocketService(
     return websocket -> {
       final SocketAddress socketAddress = websocket.remoteAddress();
       final String connectionId = websocket.textHandlerID();
+      String token = getToken(websocket);
+      LOG.trace("token {}", token);
```

Comment:
Maybe we should add a bit more of context in this trace msg. Also, what happens when the token doesn't exist?

Reply:
And this can probably be a final variable :)

Reply:
if there is no token, and authentication is turned on, then when it tries to get permissions it will fail with "Unauthorized"

Reply:
If there is no token there is no point in logging `token null`, is it?

Reply:
done